src/main/java/nextstep/subway/applicaion/FavoriteService.java

```diff
@@ -0,0 +1,67 @@
+package nextstep.subway.applicaion;
+
+import nextstep.exception.CantDeleteFavoriteException;
+import nextstep.member.domain.Member;
+import nextstep.member.domain.MemberRepository;
+import nextstep.member.domain.UserDetails;
+import nextstep.subway.applicaion.dto.FavoriteRequest;
+import nextstep.subway.applicaion.dto.FavoriteResponse;
+import nextstep.subway.domain.Favorite;
+import nextstep.subway.domain.FavoriteRepository;
+import nextstep.subway.domain.Station;
+import nextstep.subway.domain.StationRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import javax.persistence.EntityNotFoundException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@Transactional
+public class FavoriteService {
+    private final MemberRepository memberRepository;
+    private final StationRepository stationRepository;
+    private final FavoriteRepository favoriteRepository;
+
+    public FavoriteService(MemberRepository memberRepository, StationRepository stationRepository,
+                           FavoriteRepository favoriteRepository) {
+        this.memberRepository = memberRepository;
+        this.stationRepository = stationRepository;
+        this.favoriteRepository = favoriteRepository;
+    }
+
+    public FavoriteResponse createFavorites(UserDetails userDetails, FavoriteRequest favoriteRequest) {
+        Member member = memberRepository.findById(userDetails.getId()).orElseThrow(EntityNotFoundException::new);
+        Station source = stationRepository.findById(favoriteRequest.getSource())
+                .orElseThrow(EntityNotFoundException::new);
+        Station target = stationRepository.findById(favoriteRequest.getTarget())
+                .orElseThrow(EntityNotFoundException::new);
+        Favorite favorite = favoriteRepository.save(new Favorite());
```

Comment:
```suggestion
        Favorite favorite = favoriteRepository.save(new Favorite(source, target));
```

setter를 사용하지 않는 방법입니다.